README.md

```diff
@@ -64,6 +67,25 @@ bind-key -n C-l if-shell "$is_vim" "send-keys C-l"  "select-pane -R"
 bind-key -n C-\ if-shell "$is_vim" "send-keys C-\\" "select-pane -l"
 ```
 
+#### Include the snippet
+
+The above code snippet is also available in the `vim-tmux-navigator.tmux`
+file, which can be includes like this in your tmux config:
+
+```tmux
+run-shell /path/to/vim-tmux-navigator/vim-tmux-navigator.tmux
+```
+
+#### TPM
+
+When using Tmux Plugin Manager ([TPM][]) add the following lines to
+`~/.tmux.conf`. For more information see [TPM][].
```

Comment:
Can we change the language here to make it more clear that this is an alternative to the "copy this snippet into your tmux.conf file" section? Something like:

> If you'd prefer, you can use the Tmux Plugin Manager ([TPM][]) instead of copying the snippet.
> When using TPM, add the following lines to your `~/.tmux.conf`